cohort_banking_initiator_js/package.json

```diff
@@ -17,6 +17,7 @@
   },
   "dependencies": {
     "@types/pg": "^8.10.2",
+    "cohort_sdk_client": "file:../cohort_sdk_client",
     "cohort_sdk_js": "file:../packages/cohort_sdk_js",
```

Comment:
If we consider only classes then this dependency is not required, but I had to add it here because package delivers not only public classes but also darwin image.

Reply:
You can add an optional dependency section in cohort_sdk_js. So based on the platform it will get the required dependency. From a consumer perspective, they only need to add cohort_sdk_client as a dependency.

https://github.com/kindredgroup/scylla/blob/feature/add-missing-dependencies/scylla_pg_js/package.json#L39-L43